src/vorta/views/repo_add_dialog.py

```diff
@@ -208,6 +211,24 @@ def password_listener(self):
             self.passwordLabel.setText(translate('utils', msg))
             return not bool(msg)
 
+    def cancel_job(self):
+        self.close()
+        try:
+            if self.job:
+                self.job.cancel()
+                logger.debug(f"Cancelled adding repo {self.values['repo_url']}")
+        except Exception as e:
```

Comment:
Which exception are you catching here?

Reply:
Um, yeah should have used if-else here :)

Reply:
Um yeah, I guess no need to add try except here 😅